homeworks/editerdelyhorvath/8_streamlit/weather_app_functions.py

```diff
@@ -0,0 +1,96 @@
+'''
+
+Weather_app.py hoz használt funkciók 
+
+'''
+
+import streamlit as st
+import requests
+import pandas as pd
+from datetime import datetime, timezone
+from db_connector import SqliteDB
+
+@st.cache_data(ttl=600) # TTL (Time To Live) 10 min
```

Comment:
ez és a get_forecast() function gyakorlatilag ugyanazt csinálja, csak az egyikben a weather endpoint-ra küldöd a request-et, a másikban a forecast-ra. Ez lehetne egy paraméter egy general function-ben, és akkor mindkettőt hibátlanul le tudnád kezelni. A cache is működne tovább, hiszen a cache a function és a megadott paraméterek kombinációjából jön létre (írtram ezzel kapcsolatban a technikai kérdések channelbe tegnap este)